index.js

```diff
@@ -1 +1,93 @@
-console.log('Hello, Node!')
+//Globals
+//call fs library
+const fs = require("fs");
+//pull in pets json
+const pets = "./data/pets.json";
+//read and encode the json
+const readPets = fs.readFileSync(pets, "utf-8");
+//turn it into an object
+const parsePets = JSON.parse(readPets);
+
+//Read
+const read = function() {
```

Comment:
Read should redo fs.readFileSync(pets, "utf-8"); and parse otherwise will always point to snapshot of original data.